<launch>

<arg name="node_name" default="pointcloud_crop_node" />
<arg name="input_topic" default="/input_topic_name" />
<arg name="output_topic" default="/output_topic_name" />
<arg name="min_x" default="-1.0" />
<arg name="min_y" default="-1.0" />
<arg name="min_z" default="-1.0" />
<arg name="max_x" default="1.0" />
<arg name="max_y" default="1.0" />
<arg name="max_z" default="1.0" />
<arg name="negative" default="false" />

<node name="$(arg node_name)" pkg="pointcloud_crop" type="pointcloud_crop_node" output="screen">
    <param name="min_x" value="$(arg min_x)" />
    <param name="max_x" value="$(arg max_x)" />
    <param name="min_y" value="$(arg min_y)" />
    <param name="max_y" value="$(arg max_y)" />
    <param name="min_z" value="$(arg min_z)" />
    <param name="max_z" value="$(arg max_z)" />
    <param name="negative" value="$(arg negative)" />

    <remap from="~/input" to="$(arg input_topic)" />
    <remap from="~/output" to="$(arg output_topic)" />
</node>

</launch>
